### PR TITLE
Fix missing network when value is empty

### PR DESCRIPTION
**Pull request by fernandog**
default value is '':
https://github.com/pymedusa/Medusa/blob/develop/medusa/tv.py#L1101

Should we change default to None?

@medariox @p0psicles

### CODE REVIEW

File: medusa/network_timezones.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def get_network_timezone(network, _network_dict):
     # Get the name of the networks timezone from _network_dict
     network_tz_name = _network_dict[network] if network in _network_dict else None
 
-    if network_tz_name is None and network not in missing_network_timezones:
+    if network_tz_name is None and network != '' and network not in missing_network_timezones:
```
Comment (ratoaq2) [inline medusa/network_timezones.py]: Why not? - >

```
    if not network_tz_name and network and network not in missing_network_timezones:
```

Comment (fernandog) [inline medusa/network_timezones.py]: @ratoaq2 default value for network is ''. imo it should be None so we don't need to change this code

Comment (medariox) [inline medusa/network_timezones.py]: @fernandog https://docs.python.org/2/library/stdtypes.html#truth-value-testing

**Comment by fernandog** [comment]
@medariox approve?

**Pull request merged**